code/src/components/AlbumCard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Overlay from './Overlay'
+import Artist from 'components/Artist'
+
+const AlbumCard = (props) => {
+  const albumsArray = props.albumsArray
+  console.log(albumsArray)
```

Comment:
remember to remove console.logs  from your code after you finish debugging your code.